warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IteratorBuildingVisitor.java

```diff
@@ -633,12 +644,6 @@ public Object visit(ASTMethodNode node, Object data) {
         return null;
     }
     
-    @Override
-    public Object visit(ASTSizeMethod node, Object data) {
```

Comment:
ASTSizeFunction

Reply:
Would we not want to visit the children of a size function node?

Reply:
I guess I have to respond with a question.  Can one of the children of the size function be an expression like maybe another function?  I am thinking of something like f:func(bla).size()

Reply:
Well, actually in the context of an IteratorBuildingVisitor, it would have to be something like:
size(((_Value_ = true) && (x == y))).

Reply:
But now that I look more closely we were not doing anything with the size function before, so nevermind.